README.md

```diff
@@ -1,22 +1,30 @@
+<!-- Centered Logo -->
 <p align="center">
-<picture>
-  <source media="(prefers-color-scheme: dark)" srcset="https://github.com/abelljs/abell/assets/30949385/0452e85b-df57-47a3-891c-33068de5c0e2">
-  <source media="(prefers-color-scheme: light)" srcset="https://github.com/abelljs/abell/assets/30949385/e9ea17d6-c248-4ef6-9803-fd058b131b4a">
   <img width="512" alt="Abell Logo and Title" src="https://github.com/abelljs/abell/assets/30949385/e9ea17d6-c248-4ef6-9803-fd058b131b4a">
-</picture>
```

Comment:
This is needed to have different logos on different themes. 

With your change, the logo becomes invisible on dark mode.

<img width="761" alt="image" src="https://github.com/abelljs/abell/assets/30949385/abd7010b-bfee-49d1-b369-3c7d12013763">